guacamole/src/main/webapp/translations/en.json

```diff
@@ -750,7 +750,7 @@
         
         "FORMAT_STARTDATE" : "@:APP.FORMAT_DATE_TIME_PRECISE",
 
-        "HELP_SESSIONS" : "All currently-active Guacamole sessions are listed here. If you wish to kill one or more sessions, check the box next to those sessions and click \"Kill Sessions\". Killing a session will immediately disconnect the user from the associated connection.",
+        "HELP_SESSIONS" : "Your currently-active Guacamole sessions are listed here, and Administrators will be able to see currently-active sessions for all users.  If you wish to kill one or more sessions, check the box next to those sessions and click \"Kill Sessions\". Killing a session will immediately disconnect the user from the associated connection.",
```

Comment:
This is certainly true for the database authentication extensions, but not necessarily true in the general case. Extensions are free to define their own systems for determining who has permission to view/kill a particular session, so long as they properly expose that permission through the relevant permission sets.

For other parts of the administrative interface, the wording of the help text both describes what may be done and that those actions depend on access level. For example, from the connection interface:

> Click or tap on a connection below to manage that connection. Depending on your access level, connections can be added and deleted, and their properties (protocol, hostname, port, etc.) can be changed.

The new wording is a step in the right direction, but I don't think we should assert that the permissions work a specific way, even if it's the most common implementation of those permissions. Perhaps similar "depending on access level", etc. wording would be useful here?

Reply:
How about:

> This page will be populated with currently-active connections.  The connections listed and the ability to kill those connections is dependent upon your access level. ...

That sound okay?